channeldb/migration30/revocation_log.go

```diff
@@ -216,6 +216,22 @@ type RevocationLog struct {
 	// HTLCEntries is the set of HTLCEntry's that are pending at this
 	// particular commitment height.
 	HTLCEntries []*HTLCEntry
+
+	// LocalBalance is the current available balance within the channel
+	// directly spendable by us. In other words, it is the value of the
+	// to_remote output on the remote parties' commitment transaction.
+	//
+	// NOTE: this is a pointer so that it is clear if the value is zero or
```

Comment:
nit: may be worth adding why this can be nil + the migration number from when they are always set (this one)

Reply:
Cool, I expanded on the comment a bit 👍 

There isnt a migration number from which they will always be set though, only a commit/tag number